homeassistant/components/light/isy994.py

```diff
@@ -69,6 +69,10 @@ def turn_on(self, brightness=100, **kwargs) -> None:
             _LOGGER.debug('Unable to turn on light.')
 
     @property
+    def state_attributes(self):
```

Comment:
The bot is wanting a comment for your new property, just like line 77 for the supported_features property.